ansible/setup-new-fyre-host-play/Jenkinsfile

```diff
@@ -26,12 +25,16 @@ timestamps {
       //def scmVars = checkout scm
       checkout scm
       stage('Stage 1') {
+        withCredentials([string(credentialsId: 'nest-ops-ansible-vault-password', variable: 'decryptionPass')]){
```

Comment:
This file feels specific to nest and should likely live in a nest specific repo.